packages/plugin-ext-vscode/src/node/scanner-vscode.ts

```diff
@@ -43,7 +43,10 @@ export class VsCodePluginScanner extends TheiaPluginScanner implements PluginSca
             },
             entryPoint: {
                 backend: plugin.main
-            }
+            },
+            iconUrl: plugin.icon && PluginPackage.toPluginUrl(plugin, plugin.icon),
```

Comment:
I've added it only for VS Code extensions, but I think it it should be useful for Theia plugins as well, i.e. if in che you have a detail widget for plugin it will be useful to fetch which data. @benoitf @azatsarynnyy Should I align Theia scanner as well?